app/src/main/java/org/mozilla/focus/browser/BrowserNavigationOverlay.kt

```diff
@@ -54,7 +54,7 @@ enum class NavigationEvent {
 
 class BrowserNavigationOverlay @JvmOverloads constructor(
         context: Context, attrs: AttributeSet? = null, defStyle: Int = 0 )
-    : LinearLayout(context, attrs, defStyle), View.OnClickListener {
+    : LinearLayout(context, attrs, defStyle), View.OnClickListener, OnUrlEnteredListener {
```

Comment:
Why do you need to implement OnUrlEnteredListener? This is just adding an unnecessary layer of interfaces.